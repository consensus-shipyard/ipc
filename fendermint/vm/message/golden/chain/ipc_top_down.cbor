a163497063a16b546f70446f776e45786563a266686569676874006a626c6f636b5f6861736880