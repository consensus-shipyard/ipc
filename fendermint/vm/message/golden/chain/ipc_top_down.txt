Ipc(TopDownExec(ParentFinality { height: 0, block_hash: [] }))